webtau-feature-testing/test-expectations/scenarios/graphql/weatherQuery/run-details.json

```diff
@@ -0,0 +1,10 @@
+{
+  "scenarioDetails" : [ {
+    "scenario" : "check weather",
+    "shortContainerId" : "weatherQuery.groovy",
+    "stepsSummary" : {
+      "numberOfSuccessful" : 3
+    }
+  } ],
+  "exitCode" : 0
+}
```

Comment:
nit: newline

Reply:
This is actually generated by the test itself on first run, not hand written.  If there are failures when you diff this vs the actual file, you'll get an additional new line diff.  Not the end of the world but annoying so I'd rather leave it as is.  In practice you'll never look at this file any way :)